### DISCUSSION TITLE
Real-Time values

### DISCUSSION THREAD

**Discussion by GBUTTIN** [acesyde/hassio_mylight_integration#115] [General]
Hello and thanks a lot for this module !

is it possible to have the values in real time? 
I would like to add a map on my dash with this information.

Thanks

**Comment by acesyde** [comment]
Hello,

Mylight provide an aggregated value every 5 minutes and to avoid too many calls on their infrastructure I limited the scraping to 15 minutes.

Moreover you can have the realtime values by using the local connection to the Enphase Inverter.

You can also use clamp meter.
